src/main/scala/java/time/Period.scala

```diff
@@ -7,7 +7,7 @@ import java.time.temporal._
 import java.{util => ju}
 
 final class Period private (years: Int, months: Int, days: Int)
-    extends ChronoPeriod with Serializable {
+    extends ChronoPeriod with TemporalAmount with Serializable {
```

Comment:
I believe that `ChronoPeriod` implements `TemporalAmount`. Therefore this would be redundant.